tools/labs/Makefile

```diff
@@ -39,6 +39,12 @@ docs:
 	cp -r $(KDIR)/Documentation/output/slides/teaching/_static $(KDIR)/Documentation/output/teaching/
 
 clean::
-	-rm -rf skels
+	find skels -type f \( -name *.mod.* -or -name *.ko -or -executable -or -name *.cmd -or -name *.o \) | xargs --no-run-if-empty rm || true
```

Comment:
You don't need to add `|| true`. You can just prefix the command with `-` (minus). See https://stackoverflow.com/a/3477400/4804196.

Reply:
It make be that we want to have executable files or object files as part of the skeleton. So maybe removing those isn't the way to go.

The best way would be to look for all `Makefile` files and call `make -C $MAKEFILE_DIR clean`, where `$MAKEFILE_DIR` is the directory where the `Makefile` exists.